RELEASE_NOTES.md

```diff
@@ -12,6 +12,11 @@ When updating, always run the following commands to update gem set and database
  - check this file for changes between your old version and the one you are updating, and do the necessary manual operations if needed
  - if running on local instance (localhost), you need to precompile assets using "rake assets:precompile" once update is done
 
+4.6.0
+-----
+
+Last Sharetribe version using Rails 3.
+
```

Comment:
Yes, you are right. However, we haven't keep track of the changes and the last version bump was a year ago, so honestly I don't have a clear picture about all the things that have changes.

I hope that this version bump will be an encouragement to the whole team to start keeping up change log and doing versioning better!